internal/k8shandler/indexmanagement/index_management.go

```diff
@@ -76,30 +76,59 @@ func newDefaultPoliciesSpec(spec *logging.RetentionPoliciesSpec) *logging.Retent
 
 	defaultSpec := &logging.RetentionPoliciesSpec{
 		App: &logging.RetentionPolicySpec{
-			MaxAge: esapi.TimeUnit("7d"),
+			MaxAge:                  esapi.TimeUnit("7d"),
+			PruneNamespacesInterval: esapi.TimeUnit("30m"),
 		},
 		Infra: &logging.RetentionPolicySpec{
-			MaxAge: esapi.TimeUnit("7d"),
+			MaxAge:                  esapi.TimeUnit("7d"),
+			PruneNamespacesInterval: esapi.TimeUnit("30m"),
 		},
 		Audit: &logging.RetentionPolicySpec{
-			MaxAge: esapi.TimeUnit("7d"),
+			MaxAge:                  esapi.TimeUnit("7d"),
+			PruneNamespacesInterval: esapi.TimeUnit("30m"),
```

Comment:
Are we trying to maintain the default interval as 15m or 30m? I thought you said 15m when we spoke.

Reply:
I said 30m for delete by query thing. It was 15m for normal index management.

Reply:
Ah ok! thanks